### DISCUSSION TITLE
Vercel Authentication doesn't seem to work

### DISCUSSION THREAD

**Discussion by latifs** [vercel/community#6310] [Feature Question] [answered]
### Question

Hi Guys,
I have an app deployed on vercel and I am not ready to share it with the world so I have the vercel authentication enabled but it seems like the app is still accessible to really anyone.
Could you point me in the right direction?
Thanks

<img width="942" alt="Screenshot 2024-03-24 at 8 47 57 AM" src="https://github.com/vercel/vercel/assets/3582034/0c3ec4ce-a9e9-45b9-9369-78a0723d13e2">

**Comment by amyegan** [comment, marked as answer]
Hi @latifs. It looks like you have it set to **Standard Protection** which means the up-to-date production domain is still publicly accessible. The ability to [protect all deployments](https://vercel.com/docs/security/deployment-protection#protect-all-deployments) is part of [Advanced Deployment Protection](https://vercel.com/docs/security/deployment-protection#advanced-deployment-protection) which is an optional add-on for Pro.

<img width="808" alt="image" src="https://github.com/vercel/vercel/assets/8247105/3382c8fc-0b17-4737-a67e-7fd6d363054a">

**Reply by latifs** [reply]
Hey @amyegan,

so I'm confused on the wording then, 
what is an `up-to-date-Production Domain` then and how could I make it `not up to date`, I guess.

Especially since it says here that it is available on all plans

<img width="841" alt="image" src="https://github.com/vercel/community/assets/3582034/dcaac8ed-6c92-401e-9fa3-57a73fce5ac5">


Thanks

**Reply by amyegan** [reply]
The up-to-date production domains are the ones that are automatically pointed at your latest production deployment every time a new production deployment is made. If I had a project using `my-awesome-app.vercel.app`, then that would be one of my up-to-date production domains. If I assigned a [custom domain](https://vercel.com/docs/projects/domains/add-a-domain#add-and-configure-domain), then that would also be one of my up-to-date production domains. 

With Standard Protection enabled, the [generated URLs](https://vercel.com/docs/deployments/generated-urls#) for specific deployments are not publicly accessible. If you hover over the "up-to-date Production Domains" text with the dotted underline you should see which URLs will still be publicly accessible.

<img width="486" alt="" src="https://github.com/vercel/community/assets/8247105/3245a958-7f6a-4a09-beb6-20d86b93c828">

If you want to prevent anyone from seeing your full site before it's ready, you could make a small production deployment with a single page letting visitors know the site is under construction. You could use a [custom production branch](https://vercel.com/docs/deployments/git#customizing-the-production-branch) or [disable automatic git deployment](https://vercel.com/docs/projects/project-configuration/git-configuration#git.deploymentenabled) for your current production branch to prevent leaking your work in progress until you're ready to publish it.

I hope that helps! 🙏
